### DISCUSSION TITLE
GMG Davy Crockett 12v grill

### DISCUSSION THREAD

**Discussion by notsosimplelife** [nebhead/PiFire#187] [General]
My controller just died on my Green Mountain Grills Davy Crockett. This is the portable version and runs from a 12v supply. I see that most of the grills people are fitting PiFire to are 120v but I am assuming that if I just run everything as per the designs on here but have the relays switching 12v for the fan, ignitor and auger then I will be fine. Has anyone else fitted one to a Davy Crockett?  

Thanks....Justin

**Comment by jbawgs** [comment]
I imagine you'd also swap out the power supply to a 12v-5v buck converter or the like.

**Reply by notsosimplelife** [reply]
Thanks, just done that and ordered all the parts from Ali Express.
